_client/src/Ui/Views/BaseView.hpp

```diff
@@ -1,13 +1,15 @@
 #pragma once
 
 #include <QWidget>
+#include <QGroupBox>
 #include <QVBoxLayout>
 #include <QHBoxLayout>
 #include <QPushButton>
 #include <QTableWidget>
 #include <QLabel>
 #include <QSpacerItem>
 #include "Ui/Views/ViewStyles.hpp"
```

Comment:
separate includes according to the rules (new line between them)
1. std includes
2. third-party libraries
3. our code